src/style.css

```diff
@@ -0,0 +1,134 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+.html {
+  font-family: Avenir, sans-serif;
+}
+
+.label.searchbar {
+  width: 100%;
+  height: 70px;
+}
+
+.searchbar--big {
+  position: relative;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  margin-top: 20px;
+}
+
+.icon {
+  position: absolute;
+  z-index: 1;
+}
+
+.icon--big {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.icon--small {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.input--big {
+  position: absolute;

```

Comment:
do you need the `position: absolute`? try to remove it and fix the styles without 
```suggestion
```

Reply:
Yep, remove `position` prop at all